src/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__title {
+    color: #34568b;
```

Comment:
Consider using a variable for the hover color `#34568b` to maintain consistency with other color definitions.